README.md

```diff
@@ -6,12 +6,118 @@ output:
+
+#### Underlying frameworks
+[periscope](https://cran.r-project.org/web/packages/periscope/index.html) is depending on bootstrap 3 supported shiny packages:
+
+- [shinydashboard](https://cran.r-project.org/web/packages/shinydashboard/index.html)
+- [shinydashboard Plus](https://cran.r-project.org/web/packages/shinydashboardPlus/index.html)
+- [shinyBS](https://cran.r-project.org/web/packages/shinyBS/index.html)
+
+While periscope2 is depending only on bootstrap 4 supported package [bs4Dash](https://cran.r-project.org/web/packages/bs4Dash/index.html) to cover the same functions that are presented by [periscope](https://cran.r-project.org/web/packages/periscope/index.html) dependencies and more (i.e dashboard functionality, alerts, tool-tips, etc..) in more modernized look and feel
+
+----
+
+#### Generated applications layout
+[periscope](https://cran.r-project.org/web/packages/periscope/index.html) full generated app layout is simple as follow:
+
+- Left sidebar
+- Right sidebar
+- Header
+- Body
+
+All above parts are customizable except for header, user can only update title part in it
+
+However, periscope2 full generated app layout has the following structure:
+
+- Left sidebar
+- Right sidebar
+- Header
+- Footer
+- Body
+
+For periscope2, each part has it own UI R file that can customized and updated alone. Aslo, each part has its own dedicated alert panel except for header that has an additional alert panel for announcements framework.
+
+User can adapt layout for both packages generated apps easily via related functions (refer to `r ?periscope` and `r ?periscope2` for more info)
+
+-----
+
+#### Generated applications code structure
+
+- Regardless files contents, both packages generated applications share the same first level folder structure:
+
+<center><img src="./vignettes/figures/App_Structure.png"></center>
+
+- After that, there are differences in next level structure as follow:
+  - **program** folder, that contains application main logic, periscope2 has:
+    - a dedicated UI R script for each layout part to control it
+    - **config**  - folder for storing application different configuration files (it will evolve more in next versions)
+      - Initially contains announcements module configuration file
+    - **modules**  - folder for application shiny modules R scripts (it will evolve more in next versions)
+    <center><img src="./vignettes/figures/Program_Folder.png"></center>
+  - ***www*** folder that contains the application JS, CSS and theme customization files
+    - Although generated applications by both frameworks will have the same folder structure, periscope2 generated applications will have two additional files **css/custom.css** and **js/custom.js** to have application custom JS and CSS code
+      - Note that both files are already linked to generated app and ready to be used immediately by user
```

Comment:
change slightly to:
"- Both of these files are already linked into the generated app and ready to be used"